modules/exploits/windows/local/unquoted_service_path.rb

```diff
@@ -30,17 +32,22 @@ def initialize(info={})
 
         The offensive technique is also described in Writing Secure Code (2nd Edition),
         Chapter 23, in the section "Calling Processes Security" on page 676.
+
+        This technique was previously called Trusted Service Path, but is more commonly
+        known as Unquoted Service Path.
       },
       'References'     =>
         [
           ['URL', 'http://msdn.microsoft.com/en-us/library/windows/desktop/ms682425(v=vs.85).aspx'],
-          ['URL', 'http://www.microsoft.com/learning/en/us/book.aspx?id=5957&locale=en-us']  #pg 676
+          ['URL', 'http://www.microsoft.com/learning/en/us/book.aspx?id=5957&locale=en-us'],  #pg 676
+          ['URL', 'https://medium.com/@SumitVerma101/windows-privilege-escalation-part-1-unquoted-service-path-c7a011a8d8ae']
         ],
       'DisclosureDate' => "Oct 25 2001",
       'License'        => MSF_LICENSE,
       'Author'         =>
         [
-          'sinn3r'
+          'sinn3r', #msf module
+          'h00die'  #improvements
         ],
       'Platform'       => [ 'win'],
       'Targets'        => [ ['Windows', {}] ],
```

Comment:
It looks like this module pre-dated the `Notes` metadata, but while you're in here could you please add them?

Based on the behavior, I'm thinking:
```ruby
      'Notes'          =>
        {
          'Stability'   => [ CRASH_SERVICE_DOWN ],
          'SideEffects' => [ ARTIFACTS_ON_DISK, CONFIG_CHANGES ],
          'Reliability' => [ REPEATABLE_SESSION, ],
        },
```